src/main/java/com/nerodesk/om/mock/MkFriends.java

```diff
@@ -83,16 +97,19 @@ public boolean leader() throws IOException {
         assert this.dir != null;
         assert this.user != null;
         assert this.label != null;
```

Comment:
@pinaf yes, they were here but what is purpose now?. Lets move this check to constructor or maybe it's better to add `@NotNull` for contructor's parameters. What do you think?